sound/soc/sof/intel/hda.c

```diff
@@ -542,6 +542,10 @@ int hda_dsp_probe(struct snd_sof_dev *sdev)
 	hdev->no_ipc_position = sof_ops(sdev)->pcm_pointer ? 1 : 0;
 #endif
 
+	ret = pci_enable_device(pci);
+	if (ret < 0)
+		return ret;
+
```

Comment:
@plbossart Hmm, we are doing pcim_enable_device() already in sof_pci_probe(), and this would seem like the more appropriate place (this is not Intel/hw specific after all, right). Am I looking at wrong place?

Reply:
Ah nice catch @kv2019i 
I was looking heads-down for changes with the SKL driver and missed this.
Ok, so this patch is useless. That was confirmed in practice, but now I know why :-)